### PR TITLE
publish: posting links with report/showcase to PR

### PR DESCRIPTION
**Pull request by qfox**
ref #1207

---

Showcase is [available](http://bem.github.io/reports/5728/showcase/showcase.html)

### CODE REVIEW

**Comment by SevInf** [comment]
Showcase

**Comment by qfox** [comment]
@SevInf http://bem.github.io/reports/4354/showcase/showcase.html ;-)

**Comment by SevInf** [comment]
Showcase

**Comment by SevInf** [comment]
:tada: Showcase is available at http://bem.github.io/reports/4361/showcase/showcase.html

File: post_comment.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#/usr/bin/sh
+
+[ -z "$TRAVIS_REPO_SLUG" ] && echo "No TRAVIS_REPO_SLUG provided" && exit 1
+[ -z "$TRAVIS_PULL_REQUEST" ] && echo "No TRAVIS_PULL_REQUEST provided" && exit 2
```
Comment (qfox) [inline post_comment.sh]: Feels like these conditions are wrong ;-(

**Comment by SevInf** [comment]
May we please don't use me as a bot?:)

**Comment by qfox** [comment]
@SevInf Sorry, it's all about GH_TOKEN used at travis ;-).

This PR is proposal, so we should change it if we would work here.

@tadatuta If we don't need posting links to PRs — need to take pushlish\* scripts updates. Seems like there is a race troubles combined with caching. You can see that a lot of showcase folders was created at the same time after I've added `rebase`: https://github.com/bem/reports/ I have no explanation of that.

**Comment by tadatuta** [comment]
@zxqfox :+1: 
I like the idea to publish link into PR but maybe it's possible [to update pull request](https://developer.github.com/v3/pulls/#update-a-pull-request) itself instead of adding comment?

**Comment by qfox** [comment]
@tadatuta Yes, I thought about it, like bountysource.
It's a little bit harder:
- requires 2 requests (read + write, as I see in API);
- requires check for content + append or patch (cleanup + append) original content;
- and looks like it's easier to make in js instead of shell.

Let's file an issue about it to clarify details ;-) #1331

**Comment by aristov** [comment]
ping

**Comment by qfox** [comment]
@aristov Do you wanna tackle this? ;-)

**Comment by aristov** [comment]
@zxqfox I don't know anything about shell. Just wanna know is there any progress on this in the context of common PRs review.

**Comment by tadatuta** [comment]
I'll take it. Will make an update and merge tomorrow.

**Comment by narqo** [comment]
@tadatuta ping

**Comment by SevInf** [comment]
:tada: Showcase is available at http://bem.github.io/reports/5692/showcase/showcase.html

**Comment by birhoff** [comment]
Sorry for some spam, I just up to date branch, and test that all is work. In the near future change token on BOT and add url to description instead of commenting.

**Comment by birhoff** [comment]
up, ready for review:
- report and showcase posted only on PR travis build
- report posts only if gemini fail
- before specs tests clean notifications

**Pull request merged**